external-crates/move/crates/move-package/resolve_pkg_name_nested_dep_conflict/deps_only/C/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "C"
+
+[dependencies.A]
+resolver = "../../../resolvers/successful_subst.sh"
```

Comment:
(I think the problem is that the `ADep` name is coming from the external resolver which doesn't provide an `ADep-renamed`, only an `ADep`).